docs/ru/query_language/table_functions/mysql.md

```diff
@@ -0,0 +1,72 @@
+# mysql
+
+Allows to perform `SELECT` queries on data that is stored on a remote MySQL server.
```

Comment:
Doesn't look like Russian language.

Reply:
It's just a placeholder for the future Russian translation.